pkg/slack/slack.go

```diff
@@ -56,12 +56,42 @@ func (b *Bot) Start() {
 			logging.Logger.Debug("Connection Info: ", ev.Info)
 
 		case *slack.MessageEvent:
-			// Serve only if mentioned
-			if !strings.HasPrefix(ev.Text, "<@"+botID+"> ") {
+			// Check if message posted in a channel
+			chanInfo, _ := api.GetChannelInfo(ev.Channel)
+
+			// Check if message posted in a group
+			groupInfo, _ := api.GetGroupInfo(ev.Channel)
+
+			if chanInfo != nil || groupInfo != nil {
+				// Message posted in a channel/group
+				// Serve only if starts with mention
+				if !strings.HasPrefix(ev.Text, "<@"+botID+"> ") {
+					continue
+				}
+				logging.Logger.Debugf("Slack incoming message: %+v", ev)
+				msg := strings.TrimPrefix(ev.Text, "<@"+botID+"> ")
```

Comment:
Isnt this code same as line 92 to 101. Let try to merge both. In case of channel without prefix we return or continue